.github/workflows/ossar-scan.yml

```diff
@@ -59,6 +59,6 @@ jobs:
       id: ossar
 
     - name: Upload results to Security tab
-      uses: github/codeql-action/upload-sarif@ec3cf9c605b848da5f1e41e8452719eb1ccfb9a6
```

Comment:
Why was this changed? Same question for the other changes in this PR?

Reply:
These are the changes from dependabot automatic PRs. Once they are merged, these changes will not show. 
Automatic PRs that I merged from dependabot can be found [here:](https://github.com/delaramamiri/ebpf-for-windows/pulls?q=is%3Apr+is%3Aclosed)